java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -282,8 +281,11 @@ abstract class AbstractArcHost(
 
         // All particles have now received their onStart events. Trigger any proxy sync
         // requests so that the ensuing onReady events will fire after this point.
+        val dispatcher = SchedulerDispatcher(schedulerProvider(partition.arcId))
         context.entityHandleManager.initiateProxySync()
-        context.particles.values.forEach { it.notifyWriteOnlyParticles() }
+        context.particles.values.forEach {
```

Comment:
I think this change is in your other PR, plus dispatcher = schedulerProvider(partition.arcId).asCoroutineDispatcher()  :)

Reply:
whoops, I'll have to handle the conflict... cest la vie when working with `git cherry-pick`